invisible_cities/detsim/sensor_utils.py

```diff
@@ -14,6 +15,27 @@
 from .. io      .mcinfo_io import get_sensor_binning
 
 
+def get_rate_safe(rate: float) -> float:
+    """
+    Get rate value safely, and if it is zero, raise a warning.
+
+    Parameters
+    ----------
+    rate : float
+           Value of the rate.
+
+    Returns
+    -------
+    rate if rate != 0, else rate = 0.5
+    """
+    if rate == 0:
+        warnings.warn("Zero rate is unphysical, using default "
+                      "rate of 0.5 Hz instead")
+        return 0.5 / units.ms
```

Comment:
The function output and the warning are inconsistent. Is it 0.5 Hz or 0.5 kHz? Remember that `1 Hz = 1/s`.